initial React version of the website and docs

arielmirra opened the pull request:
This PR matches the current stg.a.ml website and adds the AML/AMF documentation. Everything is implemented in React framework using Docusaurus library.

jstoiko reviewed (changes requested):
A few things (aside from my other comments):
- [ ] docker/docker compose does not seem to work
- [ ] site cannot (npm/yarn) start from `/` (setting `baseUrl` in `website/docusaurus.config.js` to `/` instead of `/amf-documentation/` break all images)
- [ ] when (npm run) build-ing the site and then (npm run) serve-ing it as is, it won't work unless changing the `baseUrl` in `website/docusaurus.config.js` to `/`

arielmirra commented on website/docusaurus.config.js:
```diff
@@ -0,0 +1,107 @@
+    title: 'AML',
+    tagline: 'Standards and tools to define, parse and link metadata',
+    url: 'https://aml-org.github.io',
+    baseUrl: '/amf-documentation/',
```
can't seem to reproduce the issue with images, I've updated the baseUrl to `'/'` and everything works as intented.

arielmirra commented:
> A few things (aside from my other comments):
> 
> * [ ]  docker/docker compose does not seem to work
> * [ ]  site cannot (npm/yarn) start from `/` (setting `baseUrl` in `website/docusaurus.config.js` to `/` instead of `/amf-documentation/` break all images)
> * [ ]  when (npm run) build-ing the site and then (npm run) serve-ing it as is, it won't work unless changing the `baseUrl` in `website/docusaurus.config.js` to `/`
> * [x]  seems like there is a default `/blog/` subfolder being built but it's empty, I would remove it
> * [ ]  feels a little odd to have the root-level website source code inside a `/website` folder, I would try to move it to the root of the repo

The configuration file has been updated in the latest commit, everything should work just fine.
I've removed the unused blog folder
Regarding the folder structure, it can't be changed due to the framework being used (docusaurus), but I've added a root-level `package.json` file with two valuable scripts to simplify building:
- `npm run preview` serves the live view of the website
- `npm run build` build the static site and serves it
both run the server in localhost:3000.

I've updated the `README.md` to reflect this changes.
